test/schema/artist/index.js

```diff
@@ -18,11 +18,31 @@ describe('Artist type', () => {
       .onCall(0)
       .returns(Promise.resolve(42));
     Artist.__Rewire__('total', total);
+
+    const bio = sinon.stub();
+    bio
+      .onCall(0)
+      .returns(Promise.resolve(null));
+    Artist.__Rewire__('bio', bio);
+
+    const blurb = sinon.stub();
+    blurb
+      .onCall(0)
+      .returns(Promise.resolve(null));
+    Artist.__Rewire__('blurb', blurb);
+
+    const positron = sinon.stub();
+    positron
+      .onCall(0)
+      .returns(Promise.resolve(22));
```

Comment:
You’re returning a promise that yields an integer, however, the way you’re actually deriving the count is by counting the number of articles in the response: https://github.com/artsy/metaphysics/pull/298/files#diff-bd09821d2bea5da1533de71504fa3eb0R149, so to mimic that you should probably yield an array of N number of items instead.